src/canvas/static_canvas.class.ts

```diff
@@ -430,18 +430,16 @@ export class StaticCanvas<
 
     this.width = this.width || lowerCanvasEl.width || 0;
     this.height = this.height || lowerCanvasEl.height || 0;
+    this.viewportTransform = [...this.viewportTransform];
```

Comment:
good catch!

Reply:
i catched just because this is something that breaks with exports.
the values we put on the prototype become misteriously inaccessible, so viewportTransform is undefiend rather than the default and i have no idea why.
I m not sure if is a side effect of Object.freeze or else.
But yes this was a bug.